docs/pages/x/api/date-pickers/date-picker.json

```diff
@@ -179,9 +179,13 @@
         "description": "Array&lt;'day'<br>&#124;&nbsp;'month'<br>&#124;&nbsp;'year'&gt;"
       }
     },
+    "yearsOrder": {
+      "type": { "name": "enum", "description": "'asc'<br>&#124;&nbsp;'desc'" },
+      "default": "'asc'"
+    },
     "yearsPerRow": {
       "type": { "name": "enum", "description": "3<br>&#124;&nbsp;4" },
-      "default": "4 on desktop, 3 on mobile"
+      "default": "3"
```

Comment:
Hmm, this is not right?

Reply:
I updated the common default to cover both desktop and mobile in all cases, but when yearsPerRow was directly on the mobile pickers, the defaults were the actual value, I should probably revert the changes, but then yearsOrder should be added directly to the picker types or at least yearsPerRow will have to be added to override the ExportedYearCalendar default.

Reply:
Is this ok in the API documentation or should it be more specific for mobile?

<img width="1150" alt="MobileDateTimePicker_api_docs_default" src="https://github.com/user-attachments/assets/b57f8f51-6ac7-47df-8f13-79d9cf4dfee9">

Reply:
I think the code quotes work better than double quotes so I will update those that I noticed here:

<img width="1325" alt="Pickers_API_code_not_quotes" src="https://github.com/user-attachments/assets/e2c3cfd9-3e91-4336-a7fc-af466536f50f">